src/core/syntax.js

```diff
@@ -7,17 +7,17 @@ module.exports = async (config) => {
   const {
     temp,
     languages,
-    theme_custom,
+    syntax,
   } = config
 
   const rsh = syspath.dirname(require.resolve('react-syntax-highlighter'))
   const langs = languages
-    .filter(lang => fs.pathExistsSync(`${rsh}/languages/prism/${lang}`))
-    .map(lang => `{ name: '${lang}', func: require('${rsh}/languages/prism/${lang}').default },`)
+    .filter(lang => fs.pathExistsSync(`${rsh}/languages/${syntax.renderer}/${lang}.js`))
+    .map(lang => `{ name: '${lang}', func: require('${rsh}/languages/${syntax.renderer}/${lang}').default },`)
     .join('\n')
 
   const content = `module.exports = {
-    theme: require('${rsh}/styles/prism/${theme_custom.syntaxTheme}').default,
+    theme: require('${rsh}/styles/${syntax.renderer}/${syntax.theme}').default,
```

Comment:
Definitely don't think the user should have to specify prism or highlight. We should automatically default to one or the other depending on which one supports the language.

Reply:
I agree, but let's roll that up into the language detection epic.

Reply:
And we do default to Highlight.js now.